### ISSUE TITLE
Links to repository groups do not always work

### ISSUE THREAD

**Issue by roman-khazanskii** [gitblit/gitblit#1360]
Let's say I have a bit of an hierarchy in my repositories:

![image](https://user-images.githubusercontent.com/31830335/100672629-45309700-3373-11eb-87eb-535e43fe28d9.png)

If I click to 'Constructor', a page dedicated to this group will open. However, if I click to a nested group, I will be redirected to the main page.

Link to the 'Firmware' for example looks like this: `https://localhost:8443/project/Constructor%2FFirmware`

I ran a test on Windows 7 with Gitblit 1.9.1 GO, by the book.

**Comment by flaix** [comment]
This sounds like a duplicate of #1020. I'm marking it as duplicate, correct me if I am wrong.

**Comment by flaix** [comment]
I believe this is related to #738. 
I am not sure if this a bug or not. Actually I don't think that Gitblit has proper support for nested groups and only has the concept of a top level group. Which would make this rather an enhancement, if it is so.

**Comment by roman-khazanskii** [comment]
@flaix fair enough

**Comment by flaix** [comment]
What happens when you manually change the URL and replace the "%2F" with a "/" as described in issue 1020? What do you get?

**Comment by roman-khazanskii** [comment]
It works! Not sure what to do about it though, I just downloaded gitblit GO and ran gitblit.cmd.

**Comment by roman-khazanskii** [comment]
In cmd I see these:

```
2020-12-03 02:16:13 [INFO ] Found more URL path parts then expected, these will be ignored. Url: 'https://localhost:8443/project/Prometheus/Embedded/Schematic',
 mountpath: 'project', urlPath: 'Prometheus/Embedded/Schematic', expected 1 parameters was 3
2020-12-03 02:16:14 [INFO ] 150 repository models loaded for admin in 153 msecs
```

**Comment by flaix** [comment]
Yeah, that is what I mean. The code actually has no concept of nested groups. So this is more than a bug fix.

**Comment by roman-khazanskii** [comment]
Looks like that, yes. Than this issue is indeed a duplicate, sorry.

**Comment by flaix** [comment]
No problem. I'm happy to see that someone is actually still using Gitblit. =)
